Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:18.04 AS build
+
+RUN apt-get update && \
+    apt-get install -y \
+        build-essential \
+        git \
+        libtool \
+        libpixman-1-dev \
+        libglib2.0-dev \
+        pkg-config \
+        python
+
+WORKDIR /qemu
+
+COPY . /qemu
```

Comment:
I guess this is missing a download of the qemu source?

Reply:
this is inside the qemu source.  It'd force us to carry any changes to the build from branch to branch though.